ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-aem-cs.config

```diff
@@ -5,13 +5,19 @@ create path /content/cq:tags(sling:Folder)
 create path /etc/packages(sling:Folder)
 
 # create default contextual configuration for redirects: /conf/global/settings/redirects
-create path /conf/global(sling:Folder)/settings(sling:Folder)/redirects(nt:unstructured)
-set properties on /conf/global/settings/redirects
-  set sling:resourceType{String} to acs-commons/components/utilities/manage-redirects/redirects
-end
+create path /conf/global(sling:Folder)
+create path /conf/global/settings(sling:Folder)
+create path /conf/global/settings/redirects(nt:unstructured)
+
+# Not supported in 6.4.8
+#set properties on /conf/global/settings/redirects
+#  set sling:resourceType{String} to acs-commons/components/utilitiacs-commons-automatic-package-replicator-servicees/manage-redirects/redirects
```

Comment:
I know it's commented out, but this line got accidentally mangled in the last commit